src/Components/LandingPage/LandingPage.tsx

```diff
@@ -101,51 +101,63 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
 
   const createCheckboxLabel = () => {
     return (
-      <>
+      <div className="main-paragraph">
         <FormattedMessage
           id="disclaimer-label"
           defaultMessage="By proceeding, you confirm that you have read and agree to the "
         />
-        <Link href={getLinksForCheckbox().privacyPolicyLink} target="_blank" rel="noopener noreferrer">
+        <Link
+          href={getLinksForCheckbox().privacyPolicyLink}
+          target="_blank"
+          rel="noopener noreferrer"
+          sx={{ color: '#B85A27' }}
```

Comment:
Same thing for colors in js. We have a theme hook that you can use. You can see an example in `CategoryAccordion.tsx` on lines 21 and 34.